rudder-web/src/main/scala/com/normation/rudder/web/rest/datasource/DataSourceApiService.scala

```diff
@@ -58,7 +58,7 @@ class DataSourceApiService(
   def getSources() : Box[JValue] = {
     for {
       sources <- dataSourceRepo.getAll
-      data = sources.values.map(restDataSerializer.serializeDataSource(_))
+      data = sources.values.map(s => restDataSerializer.serializeDataSource(s._1))
```

Comment:
Status may be huge. I believe we need a "getAllDataSources"  without the status part here (and corresponding implementation of course)